packages/application-config/src/transformers.ts

```diff
@@ -19,16 +22,49 @@ const computeUriPath = (uriPath: string, entryPointUriPath: string) => {
   return `${entryPointUriPath}/${uriPath}`;
 };
 
+type TResourceAccessPermissionName<PermissionName extends string = ''> =
+  | `view${Capitalize<PermissionName>}`
+  | `manage${Capitalize<PermissionName>}`;
+
+type TImplicitCustomApplicationResourceAccesses<
+  PermissionName extends string = ''
+> = Record<TResourceAccessPermissionName<PermissionName>, string[]>;
+
 function transformCustomApplicationConfigToData(
   appConfig: JSONSchemaForCustomApplicationConfigurationFiles
 ): CustomApplicationData {
   validateEntryPointUriPath(appConfig);
   validateSubmenuLinks(appConfig);
 
+  const additionalResourceAccessKeyToOauthScopeMap = (
+    appConfig.additionalOAuthScopes || []
+  ).reduce(
+    (previousOauthScope, { name, view, manage }) => ({
+      ...previousOauthScope,
+      [`view${formatEntryPointUriPathToResourceAccessKey(name)}`]: view,
+      [`manage${formatEntryPointUriPathToResourceAccessKey(name)}`]: manage,
```

Comment:
Nit: instead of calling the function twice, extract the result into a variable.

Reply:
Updated [11ceeee](https://github.com/commercetools/merchant-center-application-kit/pull/2822/commits/11ceeee3115d57516d36fbd6193cacca1758ef99)